test/amp/amp_test.go

```diff
@@ -141,11 +271,24 @@ func (t *AmpDestinationTestRunner) validateMetric(metricName string) status.Test
 		}
 	}
 
-	// at least 2 metrics are expected with 1 set of aggregation_dimensions
-	// 1 non-aggregated + 1 aggregated minimum
-	if len(metricVals) < 2 || !foundAppendDimMetric {
-		fmt.Println("failed with less metric values than expected or missing append_dimensions")
-		return testResult
+	// AMP/Prometheus metrics do not have the appended dimensions
+	if shouldHaveAppendDimensions {
+		// at least 2 metrics are expected with 1 set of aggregation_dimensions
+		// 1 non-aggregated + 1 aggregated minimum
+		if len(metricVals) < 2 {
+			log.Println("failed with fewer metric values than expected")
+			return testResult
+		}
+
+		if !foundAppendDimMetric {
+			log.Println("failed with missing append_dimensions")
+			return testResult
+		}
+	} else {
+		if len(metricVals) < 1 {
```

Comment:
nit: Equivalent to
```suggestion
		if len(metricVals) == 0 {
```
since `len` can never be negative.

Reply:
Sure. I copied code that was `< 2` and just didnt think about changing it to `== 0`